tests/api-sessions.t

```diff
@@ -146,3 +146,11 @@ tcp,1386004309468,1386004309478,10.180.156.185,53533,US,10.180.156.249,1080,US,2
 # no map data
     $json = viewerGet("/sessions.json?startTime=1386004308&stopTime=1386004400&facets=1&expression=" . uri_escape("file=$pwd/bigendian.pcap|file=$pwd/socks-http-example.pcap|file=$pwd/bt-tcp.pcap"));
     eq_or_diff($json->{map}, from_json('{}'), "no map data");
+
+# Check file != blah.pcap
+    my $json = viewerGet("/sessions.json?date=-1&expression=" . uri_escape("file!=$pwd/https-connect.pcap"));
```

Comment:
chose several files here and do 2 tests
1) file==file1 && file==file2 && file==file3
2) file!=file1 && file==file2 && file==file3